### ISSUE TITLE
Training Mode: Starting a duel by pressing "Begin Duel" or changing to "Classic Mode" crashes the game (sometimes!)

### ISSUE THREAD

**Issue by Niklas-Eckhoff** [punyhumangames/BladeSymphony#42]
Operating system: Windows x64
Blade Symphony version: Windows x64

Maybe these two are related, but I thought it's worth it to create another issue for it: https://github.com/punyhumangames/BladeSymphony/issues/13

**Comment by Vivianite** [comment]
This issue happens upon first boot of the game and immediately going into a duel as fast as possible. the following has been attempted to reproduce it:

booting the game, waiting a minute+10 seconds into Ai opponent options before committing the change resulted in no crash.

booting the game, waiting 10 seconds+10 seconds into Ai opponent options before committing the change resulted in no crash.

booting the game, without waiting+10 seconds into Ai opponent options before committing the change resulted in no crash, however anything less has mixed results of crashing.

booting the game without waiting and without waiting on Ai opponent options before committing the change resulted in a crash.

booting the game without waiting and without waiting on Ai opponent options with default settings resulted in a crash.

Switching the setting has some degree of effect but the issue has been reproduced with default settings.

**Comment by Aquatius** [comment]
In x64 I have the same issue while switching game modes, as well as when AI opponent respawns after defeat.
